command/jsonstate/state.go

```diff
@@ -79,13 +77,11 @@ type resource struct {
 	SchemaVersion uint64 `json:"schema_version"`
 
 	// AttributeValues is the JSON representation of the attribute values of the
-	// resource, whose structure depends on the resource type schema. Any
-	// unknown values are omitted or set to null, making them indistinguishable
-	// from absent values.
-	AttributeValues attributeValues `json:"values,omitempty"`
+	// resource, whose structure depends on the resource type schema when the
+	// state was stored.
+	AttributeValues json.RawMessage `json:"values,omitempty"`
 
-	// DependsOn contains a list of the resource's dependencies. The entries are
-	// addresses relative to the containing module.
```

Comment:
This line was removed, because `DependsOn` is taking the values from the state `Dependencies` field, which is a list of resource configuration addresses, and are relative to the root module.

Reply:
This isn't a change in this PR, is it? Just a comment update? In local I'm seeing the same behaviour on the main branch and this one.

Reply:
Yup, I only removed the incorrect comment. Changing the field name to not appear to be related to `depends_on` would be a breaking change, but so there's no urgent reason to do that.